src/components/Header/LayerRoute/styles.ts

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { Button } from '@mui/material';
+
+interface IReturnRoute {
+  selectedLayer: string;
+}
+
+export const ReturnRoute = styled.div<IReturnRoute>`
+  font-size: 15px;
+  white-space: nowrap;
+  margin-left: 30px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  cursor: pointer;
+  pointer-events: auto;
+
+  .country {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'country' ? 'bold' : 'normal')};
+  }
+
+  .state {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'state' ? 'bold' : 'normal')};
+  }
+
+  .district {
+    font-weight: ${({ selectedLayer }) => (selectedLayer === 'district' ? 'bold' : 'normal')};
+  }
+
+  Button {
+    padding: 5px;
```

Comment:
```suggestion
    padding: 0px 5px;
```
for the button be more slim